app/src/main/java/com/github/se/studybuddies/ui/chat/ChatScreen.kt

```diff
@@ -288,19 +347,24 @@ fun OptionsDialog(
             Text(text = selectedMessage.getDate())
             if (viewModel.isUserMessageSender(selectedMessage)) {
               Spacer(modifier = Modifier.height(8.dp))
-              if (selectedMessage is Message.TextMessage) {
-                Button(
-                    modifier = Modifier.testTag("option_dialog_edit"),
-                    onClick = {
-                      showEditDialog.value = true
-                      showOptionsDialog.value = false
-                    }) {
-                      Text(
-                          text = stringResource(R.string.edit),
-                          style = TextStyle(color = White),
-                      )
-                    }
-                Spacer(modifier = Modifier.height(8.dp))
+              when (selectedMessage) {
+                // LinkMessage is commented due to the lack of update of the linkName field yet
+                // (only the linkUri field is updated)
+                is Message.TextMessage /*, is Message.LinkMessage*/ -> {
+                  Button(
+                      modifier = Modifier.testTag("option_dialog_edit"),
+                      onClick = {
+                        showEditDialog.value = true
+                        showOptionsDialog.value = false
+                      }) {
+                        Text(
+                            text = stringResource(R.string.edit),
+                            style = TextStyle(color = White),
+                        )
+                      }
+                  Spacer(modifier = Modifier.height(8.dp))
+                }
+                else -> {}
```

Comment:
Should this really be empty ?

Reply:
If you speak about the else -> { }, yes because I'm using a when, but not all case are defined. And I want to do nothing with the case I don't defined clearly something to do.